test/e2e/kubectl.go

```diff
@@ -111,6 +107,9 @@ var _ = Describe("kubectl", func() {
 
 })
 
+/**
+* TODO this should leverage test utils file, rather than using its own waitForGuestbookResp implementation.
```

Comment:
I don't understand this TODO. The wait here is different because it's waiting for an application specific response rather than a pod to be running or validating that a controller is in a specific state.